Public/Sdk/SelfHost/BuildXL/StandaloneTest/StandaloneTest.dsc

```diff
@@ -62,7 +62,7 @@ namespace StandaloneTest {
             `    description: "${assemblyName} [${qualifier.configuration}, ${dotNetFramework}]",`,
             `    testAssembly: f\`${def.assembly.name}\`,`,
             `    untracked: ${def.untracked},`,
-            `    wrapInDotNet: ${qualifier.targetFramework === "netcoreapp2.2"},`,

```

Comment:
>qualifier.targetFramework === "netcoreapp2.2 [](start = 33, length = 44)

Should this be changed to `BuildXLSdk.isDotNetCoreBuild` or perhaps a new property `BuildXLSdk.IsNetCoreApp`?